# David Quigley's laboratory
GitHub hosts code associated with publications from our laboratory. For additional data, including processed variant calls from some of our studies, please visit the Quigley Lab website.
 
## GitHub-hosted datasets

### [Quigley et al. Cell 2018](https://pubmed.ncbi.nlm.nih.gov/30033370)

We performed integrated analysis of DNA and RNA from 101 metastatic prostate cancer whole genomes. We show an enhancer region affects the Androgen Receptor in approximately 80% of mCRPC and demonstrated how DNA repair defects associated with loss of BRCA2, CDK12, and TP53 are associated with genomic scars of deletions, tandem duplication, and chromothripsis.  

[GitHub repository](https://github.com/DavidQuigley/WCDT)
 
### [Zhao et al. Nature Genetics 2020](https://pubmed.ncbi.nlm.nih.gov/32661416)

We performed whole genome methylation analysis of 100 metastatic prostate tumors, extending our investigation of the cohort we published in (Quigley et al Cell 2018).  

[GitHub repository](https://github.com/DavidQuigley/WCDT_WGBS)
 
### [Sjöström et al. Cancer Research 2022](https://pubmed.ncbi.nlm.nih.gov/36251389)

We performed 5-hydroxymethylcytosine analysis of 100 metastatic prostate tumors, extending our investigation of the cohort we published in (Quigley et al Cell 2018).  

[GitHub repository](https://github.com/DavidQuigley/WCDT_5hmc)
 
