# David Quigley's laboratory
GitHub hosts code associated with publications from our laboratory. For additional data, including processed variant calls from some of our studies, please visit the [Quigley Lab website](https://quigleylab.ucsf.edu).

We encourage others to use these data in their research! **Please refer to the project authors as the West Coast Dream Team (WCDT) collaboration rather than individual researchers when describing the WCDT data in the text of a manuscript or abstract.**
 
## GitHub-hosted reproducible analysis

### [Quigley et al. Cell 2018](https://pubmed.ncbi.nlm.nih.gov/30033370)

We performed integrated analysis of DNA and RNA from 101 metastatic prostate cancer whole genomes. We show an enhancer region affects the Androgen Receptor in approximately 80% of mCRPC and demonstrated how DNA repair defects associated with loss of BRCA2, CDK12, and TP53 are associated with genomic scars of deletions, tandem duplication, and chromothripsis.  

[GitHub repository](https://github.com/DavidQuigley/WCDT)
 
### [Zhao et al. Nature Genetics 2020](https://pubmed.ncbi.nlm.nih.gov/32661416)

We performed whole genome methylation analysis of 100 metastatic prostate tumors, extending our investigation of the cohort we published in (Quigley et al Cell 2018).  

[GitHub repository](https://github.com/DavidQuigley/WCDT_WGBS)
 
### [Sjöström et al. Cancer Research 2022](https://pubmed.ncbi.nlm.nih.gov/36251389)

We performed 5-hydroxymethylcytosine analysis of 100 metastatic prostate tumors, extending our investigation of the cohort we published in (Quigley et al Cell 2018).  

[GitHub repository](https://github.com/DavidQuigley/WCDT_5hmc)

### [Lundberg et al. Cancer Research 2023](https://pubmed.ncbi.nlm.nih.gov/37289025)

We used unbiased analysis of 134 metastatic tumor biopsies from the West Coast Dream Team collaboration to identify transcription factors that drive different subtypes of therapy-resistance disease.  

[GitHub repository](https://github.com/DavidQuigley/WCDT_subtypes)

### [Shrestha et al. Cancer Research 2024](https://pubmed.ncbi.nlm.nih.gov/38990734)

This work produced chromatin accessibility maps linked to the binding of lineage-specific transcription factors (TF) by performing ATAC sequencing on 70 mCRPC tissue biopsies integrated with transcriptome and whole genome sequencing.

[GitHub repository](https://github.com/DavidQuigley/WCDT_ATAC_mCRPC)

### [Zhang et al. Nature Cell Biology 2024](https://pubmed.ncbi.nlm.nih.gov/38871824)

This work showed Alternative Promoters (APs) become increasingly important as prostate cells evolve into metastatic tumors. Many groups (including us!) tend to focus on canonical gene promoters in genome-wide studies of mCRPC. Here we used very deep RNA-sequencing to interrogate all annotated promoters in benign, localized, and mCRPC biopsies. We found AP use changes as tumors develop therapy resistance and she linked these changes to binding activity of key disease drivers. We then linked changes in AP activity to epigenomic alterations at specific AP sites.

[GitHub repository](https://github.com/DavidQuigley/WCDT_alternative_promoter)
